.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,43 @@
+# Description
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
```

Comment:
Would prefer a table for this instead. 
This table can have below rows with their corresponding data. 

JIRA link 
Slack thread
Product spec
Tech spec/One-pager